src/main/java/Event.java

```diff
@@ -0,0 +1,25 @@
+public class Event extends Task {
+    protected String from;
+    protected String to;
+
+    public Event(String description, String eventFrom, String eventTo) {
```

Comment:
It would be good practice to include comments for public classes/method. 
Code Quality - Guideline: Comment minimally, but sufficiently (https://nus-cs2103-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html)